pkgs/applications/misc/gcstar/default.nix

```diff
@@ -0,0 +1,79 @@
+{stdenv, fetchFromGitLab, bash, perl, gtk2, perlPackages, wrapGAppsHook}:
+
+stdenv.mkDerivation rec {
+  pname = "gcstar";
+  version = "1.7.2";
+
+  src = fetchFromGitLab {
+      owner = "Kerenoc";
+      repo = "GCstar";
+      rev = "v${version}";
+      sha256 = "1vqfff33sssvlvsva1dflggmwl00j5p64sn1669f9wrbvjkxgpv4";
+    };
+
+  enableParallelBuilding = false;
+
+  perlDeps = [
+    perl
+    perlPackages.ArchiveZip
+    perlPackages.DateCalc
+    perlPackages.DateTimeFormatStrptime
+    perlPackages.Glib
+    perlPackages.Gtk2
+    perlPackages.GD
+    perlPackages.GDGraph
+    perlPackages.GDText
+    perlPackages.HTMLParser
+    perlPackages.JSON
+    perlPackages.ImageExifTool
+    perlPackages.librelative
+    perlPackages.LWPUserAgent
+    perlPackages.LWPProtocolHttps
+    perlPackages.MP3Info
+    perlPackages.MP3Tag
+    perlPackages.NetFreeDB
+    perlPackages.OggVorbisHeaderPurePerl
+    perlPackages.Pango
+    perlPackages.XMLSimple
+    perlPackages.XMLParser ];
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  buildInputs = perlDeps;
+  propagatedBuildInputs = perlDeps;
+
+  installPhase = ''
+    cd gcstar
+
+    perl install --text --prefix=$out
+
+    # Setting up the desktop item.
+    mkdir -p $out/share/applications
+    mkdir -p $out/share/pixmaps
+    cp share/applications/$pname.desktop $out/share/applications/$pname.desktop
```

Comment:
Is not that installed by default? https://gitlab.com/Kerenoc/GCstar/-/blob/dde0f1fbcc3c5b6bd68307ed2c3af0d68ea24bdf/gcstar/install#L182